### ISSUE TITLE
Springlock Not Murdering

### ISSUE THREAD

**Issue by Destrucnope** [ParadiseSS13/Paradise#27558]
### Exploit Reports

- [x] I confirm this issue is not an exploit. (Required)

### BYOND Version

515.1647

### Issue Description

The springlock module is not working currently. 

### What did you expect to happen?

It should've murdered me by the springlock thing

### What happened instead?

it didnt do anything

### Why is this bad/What are the consequences?

It's a bit dumb cause the description of the module say it's dangerous, and it look like it should be dangerous, but in the end is just a little bonus

### Steps to reproduce the issue.

Go under shower or splash yourself with water, and it will not kill you springtrap style

### When did the problem start happening?

I never saw it working (I am new to the server)

### Relevant log output/runtime error

```DM

```

**Comment by DGamerL** [comment]
Yep, springlocks are only affected by smoke sadly! Thanks for the issue report though!
